client/graph/graph.js

```diff
@@ -129,20 +14,38 @@ Template.graph.rendered = function(){
       }
     }));
   }
-
-  var chart = c3.generate({
-    bindto: "#"+this.data.graph_id,
-    data: {
-      //Will be something like tempdata form
-      onmouseover: showTransactions,
-      x: 'date',
-      xFormate: '%Y%m%d',
-      columns: transform(tempdata)
-    },
-    axis:{
-      x:{
-        type:"timeseries"
-      }
+  Meteor.call('txs',function(err) {
```

Comment:
This looks good.

Ideally, `Meteor.call('txs', ...)` will be called when a user logs in.
Once the data is available (we can set a flag or something, and use `setTimeout` to poll?) we have access to methods like getDayRange, etc.

Note that these methods will work regardless, they just won't return data.

I think this is good for now, until we try to support multiple users.